### ISSUE TITLE
error: cannot start job due to error: cannot list objects, XML syntax error on line 2: illegal character code U+0010

### ISSUE THREAD

**Issue by gilfreund** [Azure/azure-storage-azcopy#1197]
### Which version of the AzCopy was used? 
10.6.0

### Which platform are you using? (ex: Windows, Mac, Linux)
Linux

### What command did you run?
/azcopy copy https://s3.eu-west-1.amazonaws.com/_bucket_/_prefix_ https://_account_.blob.core.windows.net/aaa? **SES** ' --recursive=true --overwrite=ifSourceNewer

### What problem was encountered?
Job failed with the following:
INFO: Scanning...
INFO: Authenticating to source using S3AccessKey
INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support

Job bcccda57-d654-404c-6bc5-a147cb36c9e6 has started
Log file is located at: /home/backupadmin/.azcopy/bcccda57-d654-404c-6bc5-a147cb36c9e6.log

1.1 %, 6140 Done, 1 Failed, 155522 Pending, 8337 Skipped, 170000 Total (scanning...), 2-sec Throughput (Mb/s): 1010.6318
failed to perform copy command due to error: cannot start job due to error: cannot list objects, XML syntax error on line 2: illegal character code U+0010.

### How can we reproduce the problem in the simplest way?
Not sure, as I am not sure which file / step generates the error

### Have you found a mitigation/solution?
This is still in testing, but I am using --overwrite=ifSourceNewer, so I can continue the tests.

**Comment by katyp** [comment]
Hi! I have encountered this problem, too, and I've looked into it a little. I am not positive of this, but I think this is what's happening:

You use `minio` under the covers to list the contents of the bucket. Minio makes the list request and tries to parse the response as XML (here: https://github.com/minio/minio-go/blob/fe1f3855b146c1b6ce4199740d317e44cf9e85c2/api-list.go#L218)

There is a file in S3 that has a character that is not legal in XML, but it's returned in AWS's list response nonetheless:
https://forums.aws.amazon.com/thread.jspa?threadID=46125

So minio's XML parser complains.

*IF* this analysis is correct, I think the fix needs to be on AWS's side, but a workaround could be one of:
1) Avoid the files with that character in your copy (but I am not sure if `--exclude-pattern` would work in this case -- does it excludes those files from the bucket list, or would `azcopy` try to list the entire bucket and then just ignore those files from the list?)
2) Rename the files with the invalid characters
